pkg/api/login.go

```diff
@@ -63,7 +63,15 @@ func (hs *HTTPServer) LoginView(c *models.ReqContext) {
 	}
 
 	if redirectTo, _ := url.QueryUnescape(c.GetCookie("redirect_to")); len(redirectTo) > 0 {
-		c.SetCookie("redirect_to", "", -1, setting.AppSubUrl+"/")
+		cookie := http.Cookie{
```

Comment:
`HttpOnly: true,` missing

Reply:
I converted the line as it was before, I assume it was not needed. But I've added it.